com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -1193,9 +1211,10 @@ void SetRenderPassAttachments(CommandBuffer cmd, ScriptableRenderPass renderPass
                 else
                 {
                     // Only setup render target if current render pass attachments are different from the active ones
```

Comment:
Why are the store actions only used in this else branch?

Reply:
same as the previous comment. The other branch is for MRT/deferred

In general this optimization PR is only for the forward renderer at the moment, so I didn't do anything in MRT/deferred branches